source/extensions/common/dynamic_forward_proxy/dns_cache.h

```diff
@@ -148,6 +175,15 @@ class DnsCache {
    * @return all hosts currently stored in the cache.
    */
   virtual absl::flat_hash_map<std::string, DnsHostInfoSharedPtr> hosts() PURE;
+
+  /**
+   * Handle dns request
```

Comment:
Update doc to be more clear about what this does, also fix reset of doc comments.